### PR TITLE
Max/devstack notifier

### PR DESCRIPTION
**Pull request by maxrothman**
Fully integrate notifier into devstack and single-instance sandbox.

### CODE REVIEW

**Comment by fredsmith** [comment]
:+1:

File: playbooks/roles/notifier/defaults/main.yml [hunk 1]
```diff
@@ -46,8 +43,8 @@ NOTIFIER_EMAIL_REWRITE_RECIPIENT: ""
 NOTIFIER_LMS_URL_BASE: "http://localhost:8000"
 NOTIFIER_LMS_SECRET_KEY: "PUT_YOUR_SECRET_KEY_HERE"
 
-NOTIFIER_COMMENT_SERVICE_BASE: "http://localhost:4567"
-NOTIFIER_COMMENT_SERVICE_API_KEY: "PUT_YOUR_API_KEY_HERE"
+NOTIFIER_COMMENT_SERVICE_BASE: "http://localhost:18080"
+NOTIFIER_COMMENT_SERVICE_API_KEY: "password"
```
Comment (e0d) [inline playbooks/roles/notifier/defaults/main.yml:47]: why use PUT_YOUR_API_KEY_HERE but remove it above?

**Comment by e0d** [comment]
Small comment, :+1:

File: playbooks/roles/notifier/defaults/main.yml (hunk 1 above)
Comment (maxrothman) [inline playbooks/roles/notifier/defaults/main.yml:47]: If you look at [forum's defaults.yml](https://github.com/edx/configuration/blob/113233a88bdb2e051db98ec0c2027ea12094f28f/playbooks/roles/forum/defaults/main.yml#L25), its API key is set to `password` whereas [edxapp's](https://github.com/edx/configuration/blob/113233a88bdb2e051db98ec0c2027ea12094f28f/playbooks/roles/edxapp/defaults/main.yml#L286) is set to `PUT_YOUR_API_KEY_HERE`. I'm just matching what's already set.

**Pull request merged**